api/v1alpha3/nsxtloadbalancer_types.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+
+const (
+	// NSXTLoadBalancerFinalizer allows a reconciler to clean up
+	// resources associated with an NSXTLoadBalancer before removing
+	// it from the API server.
+	NSXTLoadBalancerFinalizer = "nsxtloadbalancer.infrastructure.cluster.x-k8s.io"
+)
+
+// NSXTLoadBalancerSpec defines the desired state of NSXTLoadBalancer.
+type NSXTLoadBalancerSpec struct {
+	// lbServiceID is the ID of the NSX-T LoadBalancer Service where virtual servers
+	// for Service Type=LoadBalancer are created
+	LoadBalancerServiceID string `json:"loadBalancerServiceID"`
```

Comment:
Hi @yastij,

Please correct the GoDoc comment on this field. The first word should match the name of the field.